starters/gatsby-starter-mdx-basic/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
We recently changed the licenses of our starters to be 0BSD (see https://github.com/gatsbyjs/gatsby/pull/25441). That change will be necessary here

Reply:
Thanks!